package/MDAnalysis/transformations/translate.py

```diff
@@ -63,8 +62,76 @@ def wrapped(ts):
             v = np.float32(vector)
             ts.positions += v
         else:
-            raise ValueError("{} vector in too short".format(vector))
+            raise ValueError("{} vector is too short".format(vector))
+        
+        return ts
+    
+    return wrapped
+
+def center_in_box(ag, center='geometry', point=None, pbc=None):
```

Comment:
What does it mean to have `pbc=None`? Is there a difference from having the default being `True` or `False`? If there is such a difference, it should be explicitly stated in the docstring; if not, then the default should be the correct boolean. At the moment, I do not now if PBC is taken into account by default or not.

Reply:
I tried it and it really makes no difference. However, the default in both center_of_mass and center_of_geometry methods is None, so I tried to keep things consistent. I added the default to the docs, however, to make things clearer for the user.

Reply:
Then you should figure out what the default means for `center_of_mass` and `center_of_geometry`. I could already see that `None` was the default. The question remains: what am I going to get if I do not specify `pbc`?

Reply:
Oh you were talking about the case when flags['use_pbc']